python/cac_tripplanner/cac_tripplanner/settings.py

```diff
@@ -0,0 +1,96 @@
+"""
+Django settings for cac_tripplanner project.
```

Comment:
Yeah, I actually do like this way and would be willing to go with it to see how it works out unless anyone has any major negatives they can think of.